src/services/git.ts

```diff
@@ -152,10 +152,9 @@ export function pushFeatureBranch(
   options: BackportOptions,
   featureBranch: string
 ) {
+  const remoteName = options.fork ? options.username : options.repoOwner;
```

Comment:
The only thing I did differently was move this to its own function so the logic exists in only one place (instead of having it exist in the [`doBackportVersions.ts`](https://github.com/sqren/backport/pull/141/files#diff-2063b1bf45693ebb17a50f8bb7e14c7bR157) file as well)

Reply:
Good idea. Done.